[Keycloak JavaScript CI] - Admin UI E2E (4, firefox) - realm_roles_test

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
  1) Realm roles test
       Accessibility tests for realm roles
         Check a11y violations on role details:
     AssertionError: Timed out retrying after 30000ms: Expected to find element: `.pf-v5-c-tabs `, but never found it.
      at _TabPage_getTab (webpack://@keycloak/keycloak-admin-ui/./cypress/support/pages/admin-ui/components/TabPage.ts:36:0)
      at clickTab (webpack://@keycloak/keycloak-admin-ui/./cypress/support/pages/admin-ui/components/TabPage.ts:16:0)
      at goToAttributesTab (webpack://@keycloak/keycloak-admin-ui/./cypress/support/pages/admin-ui/manage/clients/ClientRolesTab.ts:31:0)
      at  (webpack://@keycloak/keycloak-admin-ui/./cypress/e2e/realm_roles_test.spec.ts:240:0)
      at __webpack_modules__</setRunnable/runnable.fn (http://localhost:8080/__cypress/runner/cypress_runner.js:138202:43)
      at callFn (http://localhost:8080/__cypress/runner/cypress_runner.js:156821:21)
      at __webpack_modules__</Runnable.prototype.run (http://localhost:8080/__cypress/runner/cypress_runner.js:156808:13)
      at __webpack_modules__</create/onRunnableRun (http://localhost:8080/__cypress/runner/cypress_runner.js:162732:30)
      at finallyHandler (http://localhost:8080/__cypress/runner/cypress_runner.js:4072:23)
      at tryCatcher (http://localhost:8080/__cypress/runner/cypress_runner.js:1807:23)
      at __webpack_modules__</module.exports/Promise.prototype._settlePromiseFromHandler (http://localhost:8080/__cypress/runner/cypress_runner.js:1519:31)
      at __webpack_modules__</module.exports/Promise.prototype._settlePromise (http://localhost:8080/__cypress/runner/cypress_runner.js:1576:18)
      at __webpack_modules__</module.exports/Promise.prototype._settlePromise0 (http://localhost:8080/__cypress/runner/cypress_runner.js:1621:10)
      at __webpack_modules__</module.exports/Promise.prototype._settlePromises (http://localhost:8080/__cypress/runner/cypress_runner.js:1701:18)
      at _drainQueueStep (http://localhost:8080/__cypress/runner/cypress_runner.js:2407:12)
      at _drainQueue (http://localhost:8080/__cypress/runner/cypress_runner.js:2400:24)
      at __webpack_modules__</Async.prototype._drainQueues (http://localhost:8080/__cypress/runner/cypress_runner.js:2416:16)
      at __webpack_modules__</Async/this.drainQueues (http://localhost:8080/__cypress/runner/cypress_runner.js:2286:14)
```

https://github.com/keycloak/keycloak/actions/runs/11760560961/job/32761424052

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/js/apps/admin-ui/src/components/external-link/FormattedLink.tsx
+++ b/js/apps/admin-ui/src/components/external-link/FormattedLink.tsx
@@ -16,6 +16,10 @@ export const FormattedLink = ({
       href={href}
       target="_blank"
       rel="noreferrer noopener"
+      style={{
+        textDecoration: "underline",
+        textDecorationColor: "transparent",
+      }}
       className={isInline ? "pf-m-link pf-m-inline" : ""}
       {...rest}
     >

